spec.bs

```diff
@@ -8037,20 +8165,46 @@ into smaller number of fetches. It's a [=struct=] with the following [=struct/it
   : <dfn>length limit</dfn>
   :: A {{long}}, initially 2147483647 (the maximum value that it can hold). Describes the URL length
      limit the current batch is limited to, the smallest of the limits of the interest groups included.
+  : <dfn>key value interest groups</dfn>
+  :: An [=ordered map=] whose [=map/keys=] are [=tuples=] of ([=origin=], [=origin=]) and whose

```

Comment:
mention what the two origins are about. One for [trusted-bidding-signals-coordinator](https://pr-preview.s3.amazonaws.com/xtlsheep/turtledove/pull/1340.html#interest-group-trusted-bidding-signals-coordinator) and one for IG owner, IIUC?

Reply:
Actually it should be the joining origin and owner. Changed all related parts.